docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,416 @@
+**************************
+Using Sawtooth with Docker
+**************************
+
+
+This document guides you through the process of setting up Hyperledger Sawtooth
+for application development using Docker Compose, and then introduces basic
+Sawtooth concepts necessary for application development.
+
+After following the steps in this document, you will be able to perform the
+following tasks:
+
+* run Sawtooth
+* submit transactions
+* view blocks, transactions and state with the CLI
+* start and stop validators and transaction processors
+
+You will then be prepared for the more advanced tutorials that guide you in
+performing app development tasks, such as implementing business logic with
+transaction families and writing clients which use Sawtooth's REST API.
```

Comment:
Upon completing this tutorial, you will be prepared for the more advanced tutorials that guide you in performing app development tasks, such as implementing business logic with transaction families and writing clients which use Sawtooth's REST API.